### DISCUSSION TITLE
What if there was different types of liquids in OpenRCT2?

### DISCUSSION THREAD

**Discussion by TheClaudeQC** [OpenRCT2/OpenRCT2#20703] [Feature requests]
Let's be honest, Water is cool and all. But we don't see a lot of possibilities right now. Want to have another liquid today? Create a palette or use one that change the water colour.

But, what if. We weren't restricted by a single liquid, having like a couple of different liquids. Each having different impacts in each ride.

Here is a couple of ideas I have in my head and I would love to share the ideas.

We will start whit some changes whit the original one; I would make it having more exitement in rides that use water in general, but weakinging the stats of the ones that shouldn't be near ponds.

Next, would be a dream of many, to finally have actual lava: It would reduce exitement, increase, by a lot, the intensity and double the chances of breakdown for wooden rides, while giving a little more of exitement and intensity to steel ones.

Another example would be about acid: being a big problem, as only the path supports whiteout supports underneat would be able to cross it, anything else would have a height limit of 0m (basically unable to build on top of it), but will increase intensity and nausea for steel rides on the sides, also double the stat requirements of any wooden types.

There might be more that I'm not sure yet, but here is some examples of what if the liquids was more variable and impacted the game a lot more, than just a single one whit straightfowards stat boosts.

**Comment by spacek531** [comment]
How would you choose acid for a log flume, and can it kill people?

**Comment by ZehMatt** [comment]
As far as I'm aware there is no real ride that has acid or oil or any other liquids that could harm a guests, that is quite unrealistic to have in a theme park.

**Comment by TheClaudeQC** [comment]
> As far as I'm aware there is no real ride that has acid or oil or any other liquids that could harm a guests, that is quite unrealistic to have in a theme park.

Yes, but if some people want some wild stuff and don't necesserly care about a 100% realistic park, it could bring a nice touch. Also could make Infernal Views a bit better looking than just a recolour of the water

**Reply by spacek531** [reply]
I'm sure if you make a working version of it the devs might consider it

**Reply by ZehMatt** [reply]
I don't mind different water colors as for the rest, hard no.

**Reply by Broxzier** [reply]
They are some creative ideas, but I agree with the rest that this doesn't exactly belong in the game. Different water colours that can be configured like paint would be neat, though, but also a huge task on its own.

**Comment by 733737** [comment]
I don't think this would work because every scenario that uses acid or lava now just uses a palette swap and assumes it is aesthetic only. The burden of implementation is so great, and most players would be unhappy with this change.
